test/test-applications/integrations/dependency-libs/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <LangVersion>9.0</LangVersion>
```

Comment:
Should we include a comment here that explains why we are still using `9.0`?

Reply:
fixed 683acedaa30c29bb2a569b0947997ab72d6c38d9